target/linux/ramips/dts/mt7620a_dlink_dwr-961-a1.dts

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * D-Link DWR-961 A1 Board Description
+ * Copyright 2022 Pawel Dembicki <paweldembicki@gmail.com>
+ */
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "dlink,dwr-961-a1", "ralink,mt7620a-soc";
+	model = "D-Link DWR-961 A1";
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status: status {
+			label = "green:status";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		};
+
+		wan {
+			label = "green:wan";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "green:lan";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+		};
+
+		hidden-1 { /* hidden next to wlan5g led */
+			label = "green:hidden-1";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+
+		hidden-2 { /* hidden next to sms led*/
+			label = "green:hidden-2";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Just to check, are these hidden LEDs populated? How are they "hidden"? (e.g. not exposed to the external case)

Reply:
They are populated in LED panel, but they don't have transparent pictogram on front case.